app/Policies/BillingPolicy.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Policies;
+
+use App\Billing;
+use App\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class BillingPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any billings.
+     *
+     * @param User $user
+     * @return mixed
+     */
+    public function viewAny(User $user)
```

Comment:
nie wiem jak działają policy w laraverze ale czy czasem to nie sprawi, że każdy ma do wszystkiego dostęp a zatem metoda view() nie jest potrzebna?